src/commands/prompt/commands.ts

```diff
@@ -58,6 +64,43 @@ export function processCommand(input: string, context: CommandContext): boolean
     return true;
   }
 
+  if (command === '/save') {
+    if (context.messages.length === 0) {
+      output.outputInfo('There are no messages to save.');
+      return true;
+    }
+
+    try {
+      saveConversation(context, args);
+      return true;
+    } catch (error) {
+      output.outputError(error);
+      return true;
+    }
+  }
+
   output.outputError(`Unknown command: ${command} ${args.join(' ')}`);
   return true;
 }
+
+function saveConversation(context: CommandContext, args: string[]) {
+  const conversation = context.messages.reduce((prev, current) => {
+    return prev + `${current.role}: ${current.content}\n`;
+  }, '');
+
+  const conversationStoragePath = getConversationStoragePath();
+  const fileName = args[0]
+    ? path.parse(args[0]).name.replace(/\/.*\\/i, '')
+    : getDefaultFileName(context);
+  let filePath = path.join(conversationStoragePath, fileName);
```

Comment:
Looks like we are ignoring filepath specified by user. is that correct?

Reply:
yeah that's my mistake, gonna fix it

Reply:
fixed